### ISSUE TITLE
Add an application backend (e.g. wordpress)

### ISSUE THREAD

**Issue by maxammann** [digitalfabrik/entitlementcard#584]
As a user I want XYZ such that XYZ.

## Acceptance Criteria:
 - [x] Automatic deletion of "unused" applicaitons OR manual deletion
 - [x] GET applications (authorization)
 - [x] DELETE application (authorization)
 - [x] attachments?
 - [x] continue started applications
 - [ ] Maybe: Include wordpress as iframe?
 - [x] Endpoint for application frontend
 - [x] Upload verification
 - [ ] nginx IP blocker auf wp-json und wp-admin
 - [x] theme to plugin rewrite
 - [ ] email?
 - [ ] postfix relay (email change?)
 - [ ] recaptcha
 - [x] change process of bavaria

## Technical Tasks:
 - [ ] Task 1...

**Comment by maxammann** [comment]
@michael-markl I would close this if for each non-resolved point we have a corresponding task. Can you check if that is the case? I guess we need another task for recaptcha, email, IP blocker, continue started applications

**Comment by michael-markl** [comment]
IP block is not relevant any more afaics.
Continue started applications should work out of the box once #594 is merged.
I opened an issue for captcha #607
